src/Widgets/SliderWidget/SliderWidgetClass.ts

```diff
@@ -3,6 +3,7 @@ import { provideWidgetClass } from 'scrivito'
 export const SliderWidget = provideWidgetClass('SliderWidget', {
   attributes: {
     autoplay: 'boolean',
+    autoplayInterval: 'integer',
```

Comment:
* [ ] FYI I don't see why we should not allow float. I guess if the [recommendation](https://ux.stackexchange.com/a/107400) says 5 - 10, it could also be 7.5. Through testing marketing people may arrive at a non-integer sweet spot.

Reply:
I chose seconds as a unit intentionally. The current used library supports milliseconds, but we might have to switch the used library at some point. Finding a slider library that supports at least seconds will be easier to do, instead of milliseconds.

Reply:
I'm not saying we should support milliseconds, but fractional seconds.

Reply:
> [...], but fractional seconds.

How would you enforce this? If you take a float of seconds, one could set 0.001 seconds, which is identical to setting it to 1 millisecond. But float would not stop there but "allow" 0.0001 seconds or smaller - which would have no effect.

Reply:
> which would have no effect.

It would be rounded to 0 ms, which in turn will lead to a delay close to 0 ms, which is exactly what the user specified. But it's garbage input.
I'm not talking about garbage, but realistic input like 7.5 seconds.